src/Codeception/Module/WebDriver.php

```diff
@@ -3117,7 +3129,8 @@ public function loadSessionSnapshot($name)
         }
         
         foreach ($this->sessionSnapshots[$name] as $cookie) {
-            $this->setCookie($cookie['name'], $cookie['value'], (array)$cookie);
+            $showDebug = $cookie == end($this->sessionSnapshots[$name]);
+            $this->setCookie($cookie['name'], $cookie['value'], (array)$cookie, $showDebug);
```

Comment:
I think that instead of using this messy logic to detect the last cookie, it would be better to set `$showDebug` parameter to `false` in the loop and call `$this->debugCookies()` after loop.

Reply:
Yeah, fixed it